Android/res/values-es/strings.xml

```diff
@@ -0,0 +1,454 @@
+    <string name="flight_data">Datos de vuelo</string>
+    <string name="flight_modes_label">Modos de vuelo</string>
+    <string name="footprint">Marca</string>
+    <string name="ground_resolution">GSD</string>
+    <string name="guided_mode_warning">"Pulsa OK para entrar en el modo guiado(GUIDED). \nPara cambiar la localizacion, deja pulsado el mapa.</string>
+    <string name="ground_speed_label">Velocidad en tierra</string>
+    <string name="guided_scan_roi_set_message">Poniendo la localización seleccionada como objetivo</string>
+    <string name="help">Ayuda</string>
+    <string name="help_item_description"></string>
+    <string name="import_from_device">Importar fichero</string>
+    <string name="inner_wps">WPs cercanos</string>
+    <string name="label_advaned">Avanzado</string>
+    <string name="label_altitude_step"></string>
+    <string name="label_enter_filename">Poner nombre del fichero</string>
+    <string name="label_login">Loguear</string>
+    <string name="label_logout">Desloguear</string>
+    <string name="label_mission_item_type_no_selection">Opcion no disponible</string>
+    <string name="label_mission_item_type_selection">Selecciona TIpo Waypoint</string>
+    <string name="label_select_all">Seleccionar Todo</string>
+    <string name="label_sign_up">Registrarse</string>
+    <string name="label_steps_count">Numero de pasos</string>
+    <string name="label_vehicle_details">Detalles del vehiculo</string>
+    <string name="label_yes_no_dialog_dont_show">No mostrar otra vez</string>
+    <string name="length_label">Longitud</string>
+    <string name="locator">Historial de vuelos</string>
+    <string name="loiter_ccw">CCW</string>
+    <string name="lst_note_hint">Para añadir mas filas , pulsa enter</string>
+    <string name="menu_action_delete">Eliminar</string>
+    <string name="menu_action_multi_edit">Multi-Edit</string>
+    <string name="menu_action_reverse">Dar la vuelta</string>
+    <string name="menu_drone_setup">Configurar Drone</string>
+    <string name="menu_info_bar">Barra de informacion</string>
+    <string name="menu_map_type">Tipo de mapa</string>
+    <string name="menu_map_type_hybrid">Hibrido</string>
+    <string name="menu_map_type_normal">Normal</string>
+    <string name="menu_map_type_satellite">Vista Satelite</string>
+    <string name="menu_map_type_terrain">Terreni</string>
+    <string name="menu_search_toggle"></string>
+    <string name="metadata_range">Rango:</string>
+    <string name="metadata_units">Unidades:</string>
+    <string name="metadata_value">Valor:</string>
+    <string name="metadata_values">Valores:</string>
+    <string name="meter_unit">m</string>
+    <string name="mission_control_arm">Armar</string>
+    <string name="mission_control_auto">Auto</string>
+    <string name="mission_control_change_speed">Cambiar velocidad</string>
+    <string name="mission_control_condition_yaw_angle">Angulo objetivo</string>
```

Comment:
I'm not sure if he tries to reach a specific angle to the target, it is "angulo objetivo".
If the angle between the drone and the target then is "Angulo respecto del objetivo"

No estoy seguro , si trata de llegar a un angulo concreto respecto del objetivo , es "angulo objetivo".
Si es el angulo que forma el drone y el objetivo entonces es "Angulo respecto del objetivo"